<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthDialog</class>
 <widget class="QDialog" name="AuthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Users</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="credsLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>&amp;Credentials File</string>
     </property>
     <property name="buddy">
      <cstring>credsEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="newButton">
     <property name="toolTip">
      <string>Create a new username and password file.</string>
     </property>
     <property name="text">
      <string>Create &amp;New</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="addUserButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Add a new user.</string>
       </property>
       <property name="text">
        <string>&amp;Add User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteUserButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Delete the selected user.</string>
       </property>
       <property name="text">
        <string>&amp;Delete User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editUserButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Edit the selected user.</string>
       </property>
       <property name="text">
        <string>&amp;Edit User</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="discardButton">
       <property name="text">
        <string>Dis&amp;card</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="browseButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>&amp;Browse</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="credsEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Choose the file containing the list of usernames and passwords.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableView" name="usersTableView">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>credsEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>usersTableView</tabstop>
  <tabstop>addUserButton</tabstop>
  <tabstop>deleteUserButton</tabstop>
  <tabstop>editUserButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>discardButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
